### PR TITLE
MR-853 Add .destroy method

### PR DESCRIPTION
**Pull request by wavebeem**
- [x] This change is documented in CHANGELOG.md

image

### CODE REVIEW

File: src/web-sdk.tsx [hunk 1]
```diff
@@ -235,6 +237,15 @@ export type MeridianEvent = {
...
 export type MeridianMap = {
+  /**
+   * Remove the Meridian Map from the DOM and clean up all ongoing network
+   * connections.
+   *
+   * If you are writing a single page app you MUST use this call before hiding
+   * the Meridian Map, or you will having network connections that keep going in
+   * the background.
+   */
+  destroy: () => void;
```
Comment (wavebeem) [inline src/web-sdk.tsx:248]: Here's the docs

File: src/web-sdk.tsx [hunk 2]
```diff
@@ -313,6 +327,9 @@ export function createMap(
     placemarksFilter: Boolean(options.placemarks && options.placemarks.filter)
   });
   return {
+    destroy: () => {
+      render(null, element);
+    },
```
Comment (wavebeem) [inline src/web-sdk.tsx:332]: And this is the implementation. Everything else is just goodies for the demos and stuff.

File: cypress/integration/tag-response.spec.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 describe("Tag Response", () => {
-  it("visits the webSDK basic example page and select the existing tag", () => {
```
Comment (vercasson) [inline cypress/integration/tag-response.spec.ts:2]: What happened with this test, was it failing?

File: src/web-sdk.tsx [hunk 4]
```diff
@@ -235,6 +237,15 @@ export type MeridianEvent = {
  * has already been created in the page.
  */
 export type MeridianMap = {
+  /**
+   * Remove the Meridian Map from the DOM and clean up all ongoing network
+   * connections.
+   *
+   * If you are writing a single page app you MUST use this call before hiding
```
Comment (vercasson) [inline src/web-sdk.tsx:244]: I think we should add some of this additional info to the mention in the changelog. Currently the changelog info suggests to me (as a dev) that I need to go back and update all my usages to call `map.destroy()`.

File: src/web-sdk.tsx [hunk 5]
```diff
@@ -256,14 +267,17 @@ export type MeridianMap = {
...
  *  document.querySelector("#map-container"),
  *  {
+ *    api: api,
  *    locationID: "<location ID>",
- *    floorID: "<floor ID>"
+ *    floorID: "<floor ID>",
+ *    height: "500px"
  *  }
- * )
+ * );
```
Comment (vercasson) [inline src/web-sdk.tsx:278]: I would mention "single page app" here also

**Review by tarakreft — APPROVED** [review]

**Review by vercasson — APPROVED** [review]

**Pull request merged**